internal/amplitude/publish.go

```diff
@@ -0,0 +1,58 @@
+package amplitude
+
+import (
+	"bytes"
+	"net/http"
+	"time"
+
+	"github.com/cockroachdb/errors"
+	"github.com/inconshreveable/log15"
+)
+
+var AMPLITUDE_API_URL = "https://api2.amplitude.com/2/httpapi"
+
+func Publish(amplitudeAPIToken string, jsonReq []byte) error {
```

Comment:
```suggestion
func Publish(token string, body []byte) error {
```

Please also add some docstring.